src/components/dropdown/dropdown.svelte

```diff
@@ -103,7 +102,7 @@
     <slot />
   </Menu>
 </div>
-{#if showErrors && hasErrors}
+{#if showErrors && $$slots.errors}
```

Comment:
```suggestion
{#if showErrors}
```

I don't think we need the `$$slots.errors` check :smile: